src/components/Accounts/ChargeFeeFine/index.css

```diff
@@ -1,9 +1,18 @@
+@import "@folio/stripes-components/lib/variables.css";
+
 .root {  
   background-color: color(#fcfcfc contrast(100%) blend(#fcfcfc 90%));
   border-radius: 6px;
   padding: 10px 30px;
-  font-weight: bold;
-  font-size: 1rem;
-  line-height: 1.5;
+  font-weight: var(--text-weight-bold);
+  font-size: var(--font-size-medium);
+  line-height: var(--line-height);
+}
+
+.userInfoLink {
+  margin-right: var(--gutter-static-two-thirds);
 }
 
+.lastMenuButton {
+  margin: 6px;
```

Comment:
Is there a particular reason you need this extra specificity over `--gutter-static-one-third`?

Or more generally, why this class needs to be applied at all? Generally in Folio, lastMenu Buttons have no specific CSS applied and [just apply the `marginBottom0` prop](https://github.com/folio-org/stripes-smart-components/blob/7a0d94c15fee2c499581c73c34e6d0eed9468219/lib/SearchAndSort/SearchAndSort.js#L802).